packages/edit-navigation/src/components/header/menu-actions.js

```diff
@@ -63,7 +67,7 @@ export default function MenuActions( { menus, isLoading } ) {
 						className:
 							'edit-navigation-menu-actions__switcher-dropdown',
 						position: 'bottom center',
-						anchorRef: titleRef.current,
+						anchor: popoverAnchor,
```

Comment:
I'm applying the changes to this component for consistency's sake, but they can likely be ignored since the whole package may be removed soon: https://github.com/WordPress/gutenberg/issues/43620